Rakefile

```diff
@@ -58,8 +58,12 @@ namespace :build  do
     end
 
     Dir.chdir(CMAKE_WINDOWS_BUILD_FOLDER) do
+      # ensure VS commandline tools are on the path
+      VSCOMNTOOLS = ENV['VS140COMNTOOLS']
 
-      sh "cmake ../../ -G \"Visual Studio 14 2015\""
+      sh "\"#{VSCOMNTOOLS}vsvars32.bat\""
+
+      sh "cmake ../../ -DATOMIC_DEV_BUILD=0 -G \"Visual Studio 14 2015\""
```

Comment:
There is a variable for this:

https://github.com/AtomicGameEngine/AtomicGameEngine/blob/30b5ea42ecd21443d39792a273770c6ff8560de9/Rakefile#L17

Which gets cleared when packaging

https://github.com/AtomicGameEngine/AtomicGameEngine/blob/30b5ea42ecd21443d39792a273770c6ff8560de9/Rakefile#L207

It is used here:

https://github.com/AtomicGameEngine/AtomicGameEngine/blob/30b5ea42ecd21443d39792a273770c6ff8560de9/Rakefile#L79

Looks like it needs to be added to Windows, I have been running from the packaged Artifacts folder, will test that can move it once the dev build is set properly